abide/README.md

```diff
@@ -21,54 +21,59 @@ A BIDS competable version of the ABIDE dataset are avalible through [datalad rep
     You need: root of output directory, path to bids dataset, output derivative directory name.
 
     A minimal use case of this version:
-    ```
+
+    ```bash
     bash ./fmriprep_slurm_singularity_run.bash \
             ${OUTPUT_ROOT_PATH} \
             ${BIDS_DATASET_PATH} \
             derivatives
     ```
 
 3. To submit jobs for each dataset, the command to use will be similar to this.
-    ```
+
+    ```bash
     find "$OUTPUT_DIR"/.slurm/smriprep_sub-*.sh -type f | while read file; do sbatch "$file"; done
     ```
-    
+
 4. Archive the dataset. [See this page to learn the basics.](https://simexp-documentation.readthedocs.io/en/latest/alliance_canada/tape.html)
 
     The fMRIPrep data will be archived by site.
+    See `archive_fmriprep_[abide|abide2].sh`.
 
-## Timeseries extraction 
+### Use the archives
 
-The base script is `connectome_slurm_abide.bash`
-Submit the jobs through `submit_connectomes_abide.sh`.
+See `unarchive_data.sh` for fMRIPrep data and only extract data needed for the subsequent processes.
 
-## Archive other things
+## Timeseries extraction and QC
 
-```
-cd ~/scratch/abide1_connectomes-0.2.0
-tar -vczf ~/nearline/ctb-pbellec/giga_preprocessing_2/abide1_fmriprep-20.2.7lts/abide1_connectomes-0.2.0.tar .
-cd ~/scratch/abide1_giga-auto-qc-0.3.1
-tar -vczf ~/nearline/ctb-pbellec/giga_preprocessing_2/abide1_fmriprep-20.2.7lts/abide1_giga-auto-qc-0.3.1.tar .
-```
+The two steps can be run in parallel, but for the sake of computing time efficiency, I recommand the following procedure:
 
-## Use the archives
+### Running the QC and then the connectomes
 
-fMRIPrep:
+1. Run the QC first with the container with the `--reindex-bids` flag. The QC tool with create a `layout_index.sqlite` at the fMRIPrep derivative directory.
+    You use the same path binding as the input (fMRIPrep derivative directory) for the connectome tool, so the `layout_index.sqlite` created will be used correctly.
 
-```bash
-# extract file to the scratch
+2. Run the connectome tool with the container, submit one denoising strategy for each atlas first, so the group level atlas and grey matter mask will be created first.
+
+3. After the first set of timeseries extraction, submit the rest of the jobs in parallel.
+
+### Running connectome only
 
-ARCHIVE="/nearline/ctb-pbellec/giga_preprocessing_2/abide1_fmriprep-20.2.7lts/abide1_fmriprep-20.2.7lts"
-SCRATCH_TARGET="~/scratch/abide1_fmriprep-20.2.7lts"
+If you are rerunning the connectome (QC not performed)
 
-cd $ARCHIVE
-SITES=`ls`
-mkdir $SCRATCH_TARGET
-cd $SCRATCH_TARGET
+1. Run one denoising strategy on one atlas, and use the `--reindex-bids` flag.
+2. Run one denoising strategy on the remaining atlas. Use the same path binding as the input (fMRIPrep derivative directory).
```

Comment:
I'm a bit confused here. After step 1, do I run all remaining atlases and strategies in parallel? Or do I need to run one strategy for each atlas first?

Reply:
one strategy for each atlas first.

This is for preventing the nodes locking files if they are saving to the same path (for example multiple jobs creating the grey matter mask and save to the same location)

Reply:
Ok,  with the `--reindex-bids` flag?

Reply:
We don't want to recreating the `layout_index.sqlite` in step two.